README.md

```diff
@@ -11,7 +11,7 @@ Generates STS Tokens based on SAML Assertion from Azure AD (with MFA enabled als
 
 Simply run:
 
-    $ pip install git+https://github.com/piontas/python-aada.git
+    $ pip3 install git+https://github.com/piontas/python-aada.git
```

Comment:
we should keep pip because pip3 link does not exist on every environment. Readme says you should use Python3. You should run it in virtualenv or in preconfigured python3 as a main python environment